modules/cloud-run/README.md

```diff
@@ -0,0 +1,166 @@
+# Cloud Run Module
+
+Cloud Run management, with support for IAM roles and optional Eventarc trigger creation.
+
+## Examples
+
+### Traffic split
+
+This deploys a Cloud Run service with traffic split between two revisions.
+
+```hcl
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  revision_name = "green"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
+    args    = null
+    env     = null
+    env_from = null
+    ports = null
+    resources = null
+    volume_mounts = null
+  }]
+  traffic = {
+    "blue" = 25
+    "green" = 75
+  }
+}
+# tftest:skip
+```
+
+### Eventarc trigger (Pub/Sub)
+
+This deploys a Cloud Run service that will be triggered when messages are published to Pub/Sub topics.
+
+```hcl
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
+    args    = null
+    env     = null
+    env_from = null
+    ports = null
+    resources = null
+    volume_mounts = null
+  }]
+  pub_sub_triggers = [
+    "topic1",
+    "topic2"
+  ]
+}
+# tftest:skip
+```
+
+### Eventarc trigger (Audit logs)
+
+This deploys a Cloud Run service that will be triggered when specific log events are written to Google Cloud audit logs.
+
+module "cloud_run" {
+  source     = "../../modules/cloud-run"
+  project_id = "my-project"
+  name       = "hello"
+  containers = [{
+    image   = "us-docker.pkg.dev/cloudrun/container/hello"
+    command = null
```

Comment:
Most of the examples only use `image`. Maybe we can have everything except `image` in a `extra_options` field?

Reply:
And we probably want to have one example showing how to use all those fields.

Reply:
I'm not sure about the extra_options thing, because the moment they need to add any of the other params they are in a similar position with extra options. I have added more examples, have a look.

Reply:
+1 on Julio's suggestion; maybe only use image / ports / mounts / options?

Reply:
separated command, args, env and env_from to a separate options var